website/search/elastic_search.py

```diff
@@ -85,6 +85,9 @@ def wrapped(*args, **kwargs):
                 if 'ParseException' in e.error:
                     raise exceptions.MalformedQueryError(e.error)
                 raise exceptions.SearchException(e.error)
+            except TransportError as e:
```

Comment:
This is a tough call. `RequestErrors` specifically indicate a 400-series error code, whereas `TransportErrors` can be less specific.

I agree the block seems more redundant- but I also feel that the true fix for this ticket would involve understanding why we get an incorrect error code at all. That will probably involve dealing with server logs, though.

Not sure if it was the right choice, but I wrote this code as a separate block so it would be easier to remove cleanly later.